Doc/whatsnew/3.14.rst

```diff
@@ -771,6 +771,92 @@ asyncio
 * Removed implicit creation of event loop by :func:`asyncio.get_event_loop`.
   It now raises a :exc:`RuntimeError` if there is no current event loop.
   (Contributed by Kumar Aditya in :gh:`126353`.)
+  There's a few patterns that use :func:`asyncio.get_event_loop`, most
+  of them can be replaced with :func:`asyncio.run`.
+
+  If you're running an async function, simply use :func:`asyncio.run`,
+  before::
+
+    async def main():
+        ...
+
+    loop = asyncio.get_event_loop()
+    try:
+       loop.run_until_complete(amain())
+    finally:
+       loop.close()
+
+  after::
+
+    async def main():
+        ...
+
+    asyncio.run(main())
+
+  If you need to start something, e.g. a server listening on a socket
+  and then run forever, use :func:`asyncio.run` and an
+  :class:`asyncio.Event`, before::
+
+    def start_server(loop):
+        ...
+
+    loop = asyncio.get_event_loop()
+    try:
+        start_server(loop)
+        loop.run_forever()
+    finally:
+        loop.close()
+
+  after::
+
+    def start_server(loop):
+        ...
+
+    async def amain():
+        start_server(asyncio.get_running_loop())
+        await asyncio.Event().wait()
+
+    asyncio.run(amain())
+
+  If you need to run something in an event loop, then run some blocking
+  code around it, use :class:`asyncio.Runner`, before::
```

Comment:
```suggestion
  code around it, use :class:`asyncio.Runner`.
```

Reply:
This needs the :: for the code block to work